server/sonar-server/src/test/java/org/sonar/server/user/ServerUserSessionTest.java

```diff
@@ -203,6 +203,13 @@ public void hasOrganizationPermission_for_logged_in_user() {
   }
 
   @Test
+  public void hasOrganizationPermission_always_returns_true_for_roots() {
+    UserSession session = newUserSession(ROOT_USER_DTO);
+    assertThat(session.hasOrganizationPermission("an-uuid", GlobalPermissions.PROVISIONING)).isTrue();
```

Comment:
when testing an "always" condition such as here, you might want to loop over all global permissions rather than just testing a few of them (it's possible thanks to `GlobalPermissions.ALL`)